src/test/java/com/epam/tc/hw2/ex2/SelectionPageTest.java

```diff
@@ -0,0 +1,91 @@
+package com.epam.tc.hw2.ex2;
+
+import com.epam.tc.hw2.AbstractBeforeAfter;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import org.testng.annotations.Test;
+
+public class SelectionPageTest extends AbstractBeforeAfter {
+    WebDriverWait webDriverWait;
+    SoftAssertions softly = new SoftAssertions();
+    static final String EPAM_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    static final String DIFFERENT_ELEMENTS_URL = "https://jdi-testing.github.io/jdi-light/different-elements.html";
+
+    @Test
+    public void SelectionCheckTest() {
+        webDriverWait = new WebDriverWait(webDriver, 10);
+        webDriver.manage().window().maximize();
+        webDriver.navigate().to(EPAM_URL);
+        webDriverWait.until(ExpectedConditions.urlToBe(EPAM_URL));
+        //Assert Browser title
+        WebElement title = webDriver.findElement(By.name("main-title"));
```

Comment:
Browser title mean title of the page `driver.getTitle`.

Please check first tests. Also looks like unused elements